src/features/api/apiSlice.js

```diff
@@ -4,14 +4,20 @@ export const apiSlice = createApi({
   reducerPath: "api",
   baseQuery: fetchBaseQuery({ baseUrl: "https://dummyjson.com" }),
   endpoints: (builder) => ({
-    getProducts: builder.query({ query: () => "/products?limit=10" }),
+    getProducts: builder.query({
+      query: (productsToSkip) => `/products?limit=10&skip=${productsToSkip}`,
```

Comment:
catch the case of 'productsToSkip' being undefined.
it might make sense to either query the full product list if skip is not defined (=getAllProducts) or default productsToSkip to 0

Reply:
Done.